Resources/Private/CustomStylingForCkEditor/src/examplePlugin.js

```diff
@@ -0,0 +1,36 @@
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import Command from '@ckeditor/ckeditor5-core/src/command';
+
+class ExampleCommand extends Command {
+    execute(callback) {
+        const model = this.editor.model;
+        let ranges = model.document.selection._selection._ranges;
+        ranges.forEach(range => {
+            model.change(writer => {
+                writer.setAttribute('exampleAttribute', true, range);
```

Comment:
I know, you can achieve this more easily with an attributeCommand. I can do this, but then, I would suggest, I will create another example showing how a command controller gets called. I think, commandControllers will be used at least as often as attributeCommand.